app/src/main/java/com/globant/scriptsapadea/ui/activities/MainActivity.java

```diff
@@ -81,4 +98,20 @@ public void clickEdit(View view) {
     public void clickReadScript(View view) {
         Log.i("INFO", "clickReadScript");
     }
+
+    @Override
+    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+        bus.post(new ActivityResultEvent(requestCode, resultCode, data));
+    }
+
+    // TODO: Use Otto to create this event
+    @Override
+    public void onChangeFragment(Fragment fragment, boolean addInStack) {
+        if (addInStack)
```

Comment:
@leomendezg we have to ensure to encapsulate stack control inside navigator not outside because we loose control and trace if we have some problems with them.We must find other way.